examples/platforms/simulation/system.c

```diff
@@ -67,52 +68,68 @@ void otSysInit(int aArgCount, char *aArgVector[])
 {
     char *   endptr;
     uint32_t speedUpFactor = 1;
-    int      argIndex      = 0;
+
+    struct option long_options[] = {
+        {"sleep-to-tx", no_argument, 0, 's'},
+        {"time-speed-up-factor", required_argument, 0, 't'},
+    };
 
     if (gPlatformPseudoResetWasRequested)
     {
         gPlatformPseudoResetWasRequested = false;
         return;
     }
 
-    if (aArgCount < 2)
+    while (true)
     {
-        fprintf(stderr, "Syntax:\n    %s [--sleep-to-tx] NodeId [TimeSpeedUpFactor]\n", aArgVector[0]);
-        exit(EXIT_FAILURE);
-    }
+        int c = getopt_long(aArgCount, aArgVector, "", long_options, NULL);
 
-    openlog(basename(aArgVector[argIndex++]), LOG_PID, LOG_USER);
-    setlogmask(setlogmask(0) & LOG_UPTO(LOG_NOTICE));
+        if (c == -1)
+        {
+            break;
+        }
 
-    signal(SIGTERM, &handleSignal);
-    signal(SIGHUP, &handleSignal);
+        switch (c)
+        {
+        case 's':
+            gRadioCaps |= OT_RADIO_CAPS_SLEEP_TO_TX;
+            break;
+        case 't':
+            speedUpFactor = (uint32_t)strtol(optarg, endptr, 10);
+            if (*endptr != '\0' || speedUpFactor == 0)
+            {
+                fprintf(stderr, "Invalid value for TimerSpeedUpFactor: %s\n", optarg);
+                exit(EXIT_FAILURE);
+            }
+            break;
+        case '?':
+            fprintf(stderr, "Syntax:\n    %s [--sleep-to-tx] [--time-speed-up-factor=val] NodeId \n", aArgVector[0]);
```

Comment:
Both the line 106 and 116 are printing the help info. How about add an option `-h` to output the help info?